### PR TITLE
Explain usage of online Things

### LINKED ISSUE #956: Documenting online Things in Readme
Our readme does not mention the hosted Things at the Eclipse server. It is mentioned briefly at https://github.com/eclipse/thingweb.node-wot#using-a-browser but I think we should "promote" them more. This would make the server prone to possible DoS-like attacks but given that it is already on Readme, it is public enough for malicious purposes. So if we are ok with it, I would create a new section in the Readme to list these Things and what one can do with it (not detailed).

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #956

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,27 +18,44 @@ Useful labels:
...
 
--   [License](#license)
--   [Implemented/supported features](#implementedsupported-features)
-    -   [Protocol Support](#protocol-support)
-    -   [MediaType Support](#mediatype-support)
--   [Prerequisites](#prerequisites)
-    -   [To use with Node.js](#to-use-with-nodejs)
-    -   [To use in a browser](#to-use-in-a-browser)
--   [How to get the library](#how-to-get-the-library)
+-   [Eclipse Thingweb node-wot](#eclipse-thingweb-node-wot)
```
Comment (danielpeintner) [inline README.md:21]: I don't think this additional nesting is necessary.

File: README.md [hunk 2]
```diff
@@ -18,27 +18,44 @@ Useful labels:
...
+    -   [Table of Contents](#table-of-contents)
```
Comment (danielpeintner) [inline README.md:22]: I don't think we need TOC in TOC...

File: README.md [hunk 3]
```diff
@@ -18,27 +18,44 @@ Useful labels:
...
+        -   [Using a browser](#using-a-browser)
+        -   [Online Things](#online-things)
+    -   [How to use the library](#how-to-use-the-library)
+        -   [The API](#the-api)
+        -   [TD Tooling](#td-tooling)
+        -   [Logging](#logging)
+        -   [Install new/different versions of NodeJS](#install-newdifferent-versions-of-nodejs)
+    -   [Development Internals](#development-internals)
+        -   [Publishing on NPM](#publishing-on-npm)
+        -   [Regenerating package-lock.json](#regenerating-package-lockjson)
```
Comment (danielpeintner) [inline README.md:58]: I would not add internals link either..

**Review by danielpeintner — CHANGES_REQUESTED** [review]
I think the *actual* change "Online Things" is fine.
What I would mostly roll-back are the changes to the TOC

**Comment by egekorkan** [comment]
I have fixed the issue. It seems that my vs code auto generates ToC at each save...

**Review by danielpeintner — APPROVED** [review]

**Review by relu91 — APPROVED** [review]

**Pull request merged**